README.md

```diff
@@ -265,6 +272,11 @@ pip install -r requirements.txt
 pytest
 ```
 
+If you've already installed dependencies in your current virtual env, and want to update to the latest versions:
+```shell
+pip install --upgrade -r requirements.txt
+```
```

Comment:
i was getting a failed build in CI because of lint, but hadn't been getting it locally, until i updated ruff from 0.8.x to 0.9.x.